cypress/locators/commonLocators.js

```diff
@@ -6,6 +6,11 @@ const LABEL_HELP_INPUT = '#labelHelp'
 const TITLE_INPUT = '#title'
 const SUBTILE_INPUT = '#subtitle'
 const ENABLE_BACKGROUND_UI_CHECKBOX = '#enableBackgroundUI'
+const INPUT_WIDTH_SLIDER = 'input[name="inputWidth"]'
```

Comment:
Could we perhaps put the string constants in a separate file and import them in?

Reply:
I see no reason to split one common file locators into two

Reply:
structure will be fixed in next sprint as we agreed